screens/ProfileScreen.js

```diff
@@ -1,22 +1,61 @@
-import React, { Component } from "react";
-import {
-  StyleSheet,
-  Button
-} from "react-native";
+import React, { useEffect, useState } from "react";
+import { StyleSheet } from "react-native";
 import { Actions } from "react-native-router-flux";
 import { Container, Content, Header, View } from "../components/layout";
 import { Text } from "../components/typography";
+import { getUserInfo, convertToAddress } from "../utils";
+import { connect } from "react-redux";
+import theme from "../styles/base";
+
+const DEFAULT_USER_INFO = {
+  username: "",
+  email: "",
+  phone: ""
+};
+
+const DEFAULT_ADDRESS = "";
+
+const ProfileScreen = (props) => {
+  const [userInfo, setUserInfo] = useState(DEFAULT_USER_INFO);
+  const [userAddress, setUserAddress] = useState(DEFAULT_ADDRESS);
+  
+  useEffect(() => {
+    if (userInfo === DEFAULT_USER_INFO) {
+      getUserInfo(props.userId, props.token)
+        .then((response) => {
+          setUserInfo(response.data);
+          convertToAddress(response.data.location.coords, setUserAddress);
```

Comment:
I believe location could be empty if it is the user's first login. This could be undefined. Might wanna do a check inside of the function.

Reply:
You're right, the location attribute doesn't exist until they confirm location from LocationScreen. fixed.